download.html

```diff
@@ -75,59 +75,59 @@ <h3 id="stable">Version 1.3.3</h3>
 
 
 <h3 id="beta">Latest Beta Version</h3>
-<h4>Version 1.4.0b15 (20. May 2021)</h4>
+<h4>Version 1.4.0b20 (26. October 2021)</h4>
 
-Release Summary:
-<ul>
-  <li>Neural Network Classification workflow :rocket: :rocket: :rocket: see <a href="documentation/nn/nn.html">the docs</a> for more instructions </li>
-</ul>
+From 1.4.0b20 on we distribute ilastik in two versions for Windows and Linux.
+In addition to our regular build, you can download gpu-enabled builds.
+With these you can <a href="documentation/nn/nn.html">run pre-trained neural network models</a> from the <a href="https://bioimage.io/#/?partner=ilastik">bioimageio Model Zoo</a> faster, if you have a compatible nvidia GPU on your system.
+All other workflows, besides the Neural Network Workflow (local) work as in the regular build.
 
-<br>
-For more details, see the <a href="https://github.com/ilastik/ilastik/wiki/Release-Notes">Release Notes</a>.
+If you are interested in running a tiktorch inference server on a GPU machine to enable the Neural Network Workflow, please see our documentation in the <a href="https://github.com/ilastik/tiktorch">tiktorch repository</a>.
```

Comment:
How about "If you are interested in running ilastik on a _remote_ GPU, e.g. if you have a powerful server in your facility, you have to check <this doc> for additional installation instructions for the server part. After you install the server part, you can use your regular ilastik installation as a client, ilastik will take care of communicating with the server"